tests/foreman/cli/test_org.py

```diff
@@ -420,23 +351,17 @@ def test_bugzilla_1023125(self):
         graciously
 
         """
-
-        new_obj = make_org()
-
-        result = Org.info({'id': new_obj['id']})
-        self.assertEqual(result.return_code, 0, "Failed to create object")
-        self.assertEqual(len(result.stderr), 0,
-                         "There should not be an exception here")
-        self.assertEqual(new_obj['name'], result.stdout['name'])
+        org = make_org()
 
         # Create new org with the same name as before
         # should yield an exception
         with self.assertRaises(Exception):
```

Comment:
This is out of scope of this PR, though I have changed the code parts I've touched. I have created #2055 to handle this.